lib/reek/core/examiner.rb

```diff
@@ -6,7 +6,7 @@
 # See also https://github.com/troessner/reek/pull/468
 require_relative 'tree_walker'
 require_relative '../cli/warning_collector'
-require_relative '../smells/smell_repository'
+require_relative '../smells'
```

Comment:
Here then, I'd require smell_repository, since that is what Examiner uses directly.